### ISSUE TITLE
[BUG] Member stream live chat not getting

### ISSUE THREAD

**Issue by ArthurKun021** [xenova/chat-downloader#145]
## Basic information
- Program version: 0.1.10
- Python version: Python 3.9.12
- Operating system: Windows

## Describe the bug

testing a member stream with cookies, and it gives 
```python
[ERROR] Unable to parse initial video data.
```

for the member stream `1cEi9NanicE` but upon testing on other member streams in the same channel it yields no problem.
last I remeber there was a message that was posted by the channel owner before the stream started.

also did refresh the cookies but still same problem

## Command/Code used

`chat_downloader "1cEi9NanicE" --cookies "youtube.com_cookies.txt"`

### Otherwise, if using the python module, provide the following:

1. A [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example):
```python
downloader = ChatDownloader(cookies=cookies)
chat = downloader.get_chat(url=url)
```
2. Output, traceback or other information relating to the bug:
```
Retrieving chat for "None".
Unable to parse initial video data
```

## Expected behavior
will be to retrieve the stream live chat as normal

## Screenshots
![image](https://user-images.githubusercontent.com/16458204/163195646-550dd166-d227-40c8-8520-e72559ef3eae.png)

**Comment by herowinb** [comment]
I think i found the problem, Here my log with verbose
`[DEBUG] Python version: 3.10.2 (main, Jan 29 2022, 03:40:37) [GCC 10.3.1 20211027]
[DEBUG] Program version: 0.1.10
[DEBUG] Initialisation parameters: {'headers': None, 'cookies': '../config/cookies_membership.txt', 'proxy': None}
[DEBUG] Created YouTubeChatDownloader session.
[INFO] Site: youtube.com
[DEBUG] Program parameters: {'url': 'https://www.youtube.com/watch?v=7tFgcf0YcEA', 'start_time': None, 'end_time': None, 'max_attempts': 15, 'retry_timeout': None, 'interruptible_retry': True, 'timeout': None, 'inactivity_timeout': None, 'max_messages': None, 'message_groups': ['messages'], 'message_types': None, 'output': None, 'overwrite': True, 'sort_keys': True, 'indent': 4, 'format': 'youtube', 'format_file': None, 'chat_type': 'live', 'ignore': None, 'message_receive_timeout': 0.1, 'buffer_size': 4096}
[DEBUG] Starting new HTTPS connection (1): www.youtube.com:443
[DEBUG] https://www.youtube.com:443 "GET /watch?v=7tFgcf0YcEA HTTP/1.1" 200 None
[DEBUG] {'responseContext': {'serviceTrackingParams': [{'service': 'CSI', 'params': [{'key': 'c', 'value': 'WEB'},
************ I CUT THIS PART BECAUSE THEY INCLUDE HAVE MY DATA ***************
 'timestamp': {'seconds': '1649859952', 'nanos': 32385667}}}}
[WARNING] Unable to parse player response, proceeding with caution
[DEBUG] Match found: "<re.Match object; span=(0, 43), match='https://www.youtube.com/watch?v=7tFgcf0YcEA'>". Running "_get_chat_by_video_id" function in "YouTubeChatDownloader".
[DEBUG] Chat information: {'chat': <generator object YouTubeChatDownloader._get_chat_messages at 0x7f1d2209b450>, 'title': None, 'duration': None, 'status': 'past', 'video_type': 'premiere', 'start_time': None, 'id': '7tFgcf0YcEA', '_output_writer': None, '_output_callback': None, 'format': <function ChatDownloader.get_chat.<locals>.<lambda> at 0x7f1d220ea050>, 'site': <chat_downloader.sites.youtube.YouTubeChatDownloader object at 0x7f1d220c7c40>}
[INFO] Retrieving chat for "None".
[DEBUG] Getting Live chat (Live chat).
[DEBUG] https://www.youtube.com:443 "GET /live_chat_replay?continuation=********TOKEN********** HTTP/1.1" 200 None
[DEBUG] <!DOCTYPE html><html style="font-size: 10px;font-family: Roboto, Arial, sans-serif;"> **********WRONG SITE ******;</script>
[ERROR] Unable to parse initial video data. Please report this at https://github.com/xenova/chat-downloader/issues/new/choose
[DEBUG] Session closed.`
This line
`[DEBUG] https://www.youtube.com:443 "GET /`live_chat_replay`?continuation=********TOKEN********** HTTP/1.1" 200 None`
It must go to this page
`[DEBUG] https://www.youtube.com:443 "GET /`**live_chat**`?continuation=********TOKEN********** HTTP/1.1" 200 None`

**Comment by xenova** [comment]
Fixed in v0.2.0! :)
